count-with-flyers

```diff
@@ -1,17 +1,16 @@
 count_with_flyers
-stage: (reader: det), (), {}
-open_run: (None), (), {'plan_type': 'Count', 'plan_args': {'delay': '0', 'detectors': '[reader: det]', 'num': '1'}, 'detectors': ['det']}
-kickoff: (<bluesky.examples.MockFlyer object at 0x103be4978>), (), {'block_group': '_flyers'}
-wait: (None), ('_flyers',), {}
+open_run: (None), (), {'detectors': ['det'], 'plan_args': {'num': 1, 'detectors': '[reader: det]'}}
+kickoff: (<bluesky.examples.MockFlyer object at 0x10075ae10>), (), {'block_group': 'flyers-5ef9bb'}
 checkpoint: (None), (), {}
 create: (None), (), {'name': 'primary'}
-trigger: (reader: det), (), {'block_group': 'A'}
-wait: (None), ('A',), {}
+stage: (reader: det), (), {}
```

Comment:
OK, here's the case where it would cause a problem:
flyers almost certainly will depend on the detectors being staged.